mkdocs.yml

```diff
@@ -275,7 +275,8 @@ pages:
       - Key-value: platform/integration/key-value.md
       - Kubernetes: platform/integration/kubernetes.md
       - Node Cluster: platform/integration/node-cluster.md
-      - PEM Keys: platform/integration/key-pem.md
+      - PEM Keys: platform/integration/pemKey.md
+      - PEM Keys (Deprecated): platform/integration/key-pem.md
```

Comment:
There's a "Deprecated" subsection just a few lines below. That's where this needs to go.